install/mysql/glpi-empty.sql

```diff
@@ -6921,6 +6963,10 @@ CREATE TABLE `glpi_users` (
   `groups_id` int(11) NOT NULL DEFAULT '0',
   `users_id_supervisor` int(11) NOT NULL DEFAULT '0',
   `timezone` varchar(50) DEFAULT NULL,
+  `default_dashboard_central` varchar(50) DEFAULT NULL,
```

Comment:
If I am right, it will contains `glpi_dashboards_dashboards.key` value, which can be up to 100 long.
Same remark for other config options.
```suggestion
  `default_dashboard_central` varchar(100) DEFAULT NULL,
```